lib/matplotlib/axes/_axes.py

```diff
@@ -6411,16 +6411,24 @@ def hist(self, x, bins=None, range=None, density=False, weights=None,
         """
         Plot a histogram.
 
-        Compute and draw the histogram of *x*.  The return value is a tuple
-        (*n*, *bins*, *patches*) or ([*n0*, *n1*, ...], *bins*, [*patches0*,
-        *patches1*, ...]) if the input contains multiple data.  See the
-        documentation of the *weights* parameter to draw a histogram of
-        already-binned data.
-
-        Multiple data can be provided via *x* as a list of datasets
-        of potentially different length ([*x0*, *x1*, ...]), or as
-        a 2D ndarray in which each column is a dataset.  Note that
-        the ndarray form is transposed relative to the list form.
+        Computes the histogram of the data in *x* (see ~numpy.histogram) and
+        draws the results. Because the data in *x* is assumed to be unbinned,
+        bins are computed if the *bins* parameter is not set. If the histogram
```

Comment:
```suggestion
        This bins the data in *x* using `numpy.histogram` and draws the 
        freqency per bin as bar chart or polygon line. The parameters *bins*,
        *range*, *weights*, and *density* parameters are directly forwarded
        to `numpy.histogram`.
        
```
The sentence before already says "histogram", I think we can go one level deeper here and explain what actually happens.

Remove the sentence 

> The *bins*, *range*, *weights*, and *density* parameters behave as in `numpy.histogram`.

at the end of the section, because that is already included here now.



In general I would try to avoid using "histogram", because it's not quite consistent what people mean by that. Is the histogram the binned data or the plot of such data? E.g. the wikipedia (EN) describes it as "approximate representation of the frequency distribution" leaving it open whether it's data or plot, but later says "if [...] then a histogram is identical to a relative frequency plot." implying a histogram is a plot. The german wikipedia says "A histogram is a graphical representation". OTOH `numpy.histogram` is clearly only the data.